.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: Run test suite
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby-version: ['3.0', '2.7']
```

Comment:
Love this! Should we add 3.1 and 3.2 as well?

Reply:
Just added them and it works!